drivers/lora/lora_basics_modem/Kconfig

```diff
@@ -34,4 +34,13 @@ config LORA_BASICS_MODEM_RSSI_REPORT_TYPE_SIGNAL
 
 endchoice
 
+config LORA_BASICS_MODEM_DEFERRED_INIT
+	bool "Defer radio initialization until first use"
+	default n
```

Comment:
`default n` is not required, it just makes it not possible to change the default in other locations